paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -54,13 +55,13 @@ import kotlinx.coroutines.flow.filterNotNull
 import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.flow.updateAndGet
 import kotlinx.coroutines.launch
-import javax.inject.Inject
 
-internal class WalletViewModel @Inject constructor(
+internal class WalletViewModel(
```

Comment:
👀 Dagger isn't being used for this, though we should. Removed `@Inject` to avoid confusion.